Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj

```diff
@@ -7,6 +7,7 @@
     <IsPackable>false</IsPackable>
     <NoWarn>$(NoWarn),IDE0052,1573,1591,1712,CS8002</NoWarn>
     <DebugType>full</DebugType>
+    <langVersion>latest</langVersion>
```

Comment:
We already specify `<LangVersion>11.0</LangVersion>`
https://github.com/fluentassertions/fluentassertions/blob/a8a04fe96658a53c6747bfc1ea485820b1edbcb7/Directory.Build.props#L3C5-L3C5

Reply:
Removed `<langVersion>latest</langVersion> in [110ed76](https://github.com/fluentassertions/fluentassertions/pull/2307/commits/110ed76ff676dace0b57314fc71bd65f28abf5fe).